.github/workflows/propagate_to_osbuild.yml

```diff
@@ -0,0 +1,63 @@
+name: Propagate update to OSBuild
+
+on: workflow_dispatch
+
+jobs:
+  propagate:
+    runs-on: ubuntu-latest
+    env:
+      SCHUTZBOT_GH: ${{ secrets.SCHUTZBOT_GH}}
+    steps:
+      - name: Checkout manifest-db
+        uses: actions/checkout@v3
+        with:
+          path: manifest-db
+          fetch-depth: 0
+
+      - name: Update OSBuild
+        run: |
+          GH_REF="${{github.ref}}"
+          if [[ "$GH_REF" == *main* ]] ; then
+            # Get
+            MANIFEST_DB_HEAD="${{ github.sha }}"
+            echo "SHA: $MANIFEST_DB_HEAD"
+
+            # login as schutzbot
+            echo "${SCHUTZBOT_GH}" | gh auth login --with-token
+
+            # Clone osbuild
+            git clone https://github.com/osbuild/osbuild.git
+
+            cd osbuild
+            git config --local user.name "SchutzBot"
+            git config --local user.email "imagebuilder-bots+schutzbot@redhat.com"
+
+            git remote add upstream https://schutzbot:"$SCHUTZBOT_GH"@github.com/schutzbot/osbuild.git
+
+            now=$(date '+%Y-%m-%d-%H%M%S')
+            BRANCH_NAME="manifest-db-update-$now"
+            git checkout -b $BRANCH_NAME
+
+            # get the old ref from the OSBuild's Schutzfile
+            OLD_REF=$(jq -r '.global.dependencies."manifest-db".commit' Schutzfile)
+
+            # update only if the DB was updated since the version OSBuild is
+            # using
+            cd ../manifest-db
+            if [[ "$(git diff --name-only $GH_REF $OLD_REF manifest-db)" == *manifest-db* ]]; then
+              cd ../osbuild
+              # change the value for the commit head in the schutzfile
+              jq --arg variable "$MANIFEST_DB_HEAD" '.global.dependencies."manifest-db".commit=$variable' Schutzfile > Schutzfile.tmp && mv Schutzfile.tmp Schutzfile
+
+              git add -A && \
+                  git commit -m "schutzfile: update manifest-db ref" && \
```

Comment:
Maybe let's put a date (yyyy-mm-dd) into the commit message title? `rpmrepo` does that and it helps us differentiate multiple update PRs, see https://github.com/osbuild/osbuild-composer/pull/2989

Reply:
Yeah, that's quite better, especially if we happen to have multiple updates at the same time.